app/services/irs_attempts_api/envelope_encryptor.rb

```diff
@@ -18,7 +18,7 @@ def self.encrypt(data:, timestamp:, public_key:)
       formatted_time = formatted_timestamp(timestamp)
 
       filename =
-        "FCI-#{IdentityConfig.store.irs_attempt_api_csp_id}_#{formatted_time}_#{digest}.dat.gz.hex"
+        "FCI-Logingov_#{formatted_time}_#{digest}.dat.gz.hex"
```

Comment:
This was "LOGIN.gov".

The IRS wanted it to be "Logingov" because they worried that the ".gov" would be treated as a file extension. I don't think we need to accommodate a variable CSP ID, so I just hardcoded it. The "FCI-" portion is already hardcoded.